Sources/Swift/Integrations/SessionReplay/SentryTouchTracker.swift

```diff
@@ -0,0 +1,149 @@
+import Foundation
+#if (os(iOS) || os(tvOS)) && !SENTRY_NO_UIKIT
+import UIKit
+
+@objcMembers
+class SentryTouchTracker: NSObject {
+    
+    private struct TouchEvent {
+        let x: CGFloat
+        let y: CGFloat
+        let timestamp: TimeInterval
+        let phase: TouchEventPhase
+        
+        var point: CGPoint {
+            CGPoint(x: x, y: y)
+        }
+    }
+    
+    private class TouchInfo {
+        let id: Int
+        
+        var start: TouchEvent?
+        var end: TouchEvent?
+        var movements = [TouchEvent]()
```

Comment:
`l`: Again just when looking at the movements property in the code, I have no clue what it is. Maybe that's a bit cleaner:
```suggestion
        var movementToucheEvents = [TouchEvent]()
```